### PR TITLE
refactor: migrate comment style 2

### PR DESCRIPTION
**Pull request by dundargoc**
Sequel to https://github.com/neovim/neovim/pull/20012

### CODE REVIEW

**Comment by dundargoc** [comment]
Sequel to https://github.com/neovim/neovim/pull/20012

File: src/nvim/path.c [hunk 1]
```diff
@@ -835,9 +835,9 @@ static void expand_path_option(char_u *curdir, garray_T *gap)
     copy_option_part((char **)&path_option, (char *)buf, MAXPATHL, " ,");
 
     if (buf[0] == '.' && (buf[1] == NUL || vim_ispathsep(buf[1]))) {
-      /* Relative to current buffer:
-       * "/path/file" + "." -> "/path/"
-       * "/path/file"  + "./subdir" -> "/path/subdir" */
+      /// Relative to current buffer:
+      /// "/path/file" + "." -> "/path/"
+      /// "/path/file"  + "./subdir" -> "/path/subdir"
```
Comment (zeertzjq) [inline src/nvim/path.c:840]: Why three slash here?

Comment (dundargoc) [reply src/nvim/path.c:840]: Thanks

**Comment by lewis6991** [comment]
I think the regexp files are off limits since only Bram understands how they work so we keep them as close to upstream as possible.

**Comment by dundargoc** [comment]
Hmm, we already run clint on the regexp files so it's only a matter of time since all of the clint warnings are removed anyway.

**Comment by lewis6991** [comment]
Just because we run clint on those files, doesn't mean we expect the errors to be resolved at some point.

The right action (probably) is to exclude these files from clint instead of relying on the automated errors saving thing.

**Comment by dundargoc** [comment]
OK, I've removed the changes from regexp files

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**